### PR TITLE
added andreas_houben.json

### PR DESCRIPTION
**Pull request by andreashouben**
I would like to register for JS CraftCamp.

- [x] My pull request contains a JSON file `$name_or_nickname.json`
- [x] I read the `THE IMPORTANT STUFF` at [https://jscraftcamp.org/registration](https://jscraftcamp.org/registration), the JSON file follows the [template](https://github.com/jscraftcamp/website/blob/main/participants/_template.json), and contains the mandatory fields like `realName`, `githubAccountName`, `when`, `iCanTakeNotesDuringSessions`, `whatIsMyConnectionToJavascript` and `whatCanIContribute`.
- [x] If I can NOT attend, I will either send another pull request removing my JSON file or an e-mail with the subject 'UNREGISTER' to team@jscraftcamp.org
- [x] I agree that the data I enter in the registration can be used for running the event, e.g. make a name tag, count me as participant. I acknowledge that my data are public on GitHub and I agree that I will be listed on the JSCraftCamp website as a participant.
- [x] I agree that photos and videos might be taken and published (e.g. on social media) during the event.
- [x] I understand that I might NOT get a T-Shirt, because they might be in production already
- [ ] I asked my company about sponsoring (see open items at: https://github.com/orgs/jscraftcamp/projects/7)

Are you from a sponsoring company?

- [ ] Yes, I am from company ?????

### CODE REVIEW

**Comment by andreashouben** [comment]
I am a little bit confused about the sponsoring part and would like to clarify before I ask at my company. On the board there is only the "200€" part left, but there are already comments. Does that mean everyone can apply for that? How does that work? Do you have to be there to pay it if it's needed? Do you pay it up front?

**Comment by Narigo** [comment]
Welcome @andreashouben to the JSCC24! Looking forward to meeting you there and sessions about testing!

Let me answer your questions regarding the sponsoring:

> I am a little bit confused about the sponsoring part and would like to clarify before I ask at my company. On the board there is only the "200€" part left, but there are already comments from people willing to pay that. Does that mean everyone can apply for that?

Yes, you can comment on any of the sponsoring issues. If you go for the financial support, it's usually not as visible as the location, t-shirts or food sponsors, but it allows us to get either spontaneous things (ice cream?) or increase the quality of something.

> How does that work?

If your company is willing to spend money, you or your company can comment on one of the sponsoring issues or send a message to team@jscraftcamp.org to show interest. We can then 

> Do you have to be there to pay it if it's needed?

It depends on what is sponsored. For example, if you sponsor lunch, t-shirts or something else that needs to be organized up front, the sponsors sometimes organize everything themselves and pay the invoice. Sometimes, we organize it on behalf of the sponsoring company and give them the invoice to pay. We don't handle anything related to money ourselves usually.

> Do you pay it up front?

No, we use the sponsoring money more like a budget. If a company is willing to sponsor, we try to find something within these budgets. Or we ask a caterer to split their invoice into two parts so we can get two sponsors for a single item on our list, for example. We usually leave some budget available for spontaneous things like buying some ice cream - this is done by participants who bring a company credit card or have the ability to pay for it in some way and get it back from their company.

TL;DR: It depends 😅
